discovery-infra/download_logs.py

```diff
@@ -167,7 +167,7 @@ def download_logs(client: InventoryClient, cluster: dict, dest: str, must_gather
             if must_gather:
                 recreate_folder(os.path.join(output_folder, "must-gather"))
                 config_etc_hosts(cluster['name'], cluster['base_dns_domain'],
-                                 helper_cluster.get_api_vip_from_cluster(client, cluster))
+                                 helper_cluster.get_api_vip_from_cluster(client, cluster, ""))
```

Comment:
It's not understandable what's this ``""``
Can you use keyword arguments? and extract api vip into a variable